yarn.lock

```diff
@@ -21197,7 +21197,7 @@ react-devtools-core@^4.26.1, react-devtools-core@^4.28.4:
     shell-quote "^1.6.1"
     ws "^7"
 
-react-dom@18.2.0:
+react-dom@18.2.0, react-dom@^18:
```

Comment:
Don't think we want this, can we track how we have two versions installed now? Doesn't seem like it should be impacted by `micromatch`

Reply:
yeah not sure why that got added, but tried removing it and reinstalling and didn't get added back so i'll take it out